Engine/Component/ComponentScript.h

```diff
@@ -16,10 +16,16 @@ class ComponentScript :public Component
 	ComponentScript();
 	ComponentScript(GameObject* owner, std::string& script_name);
```

Comment:
If you are not going to modify the string always do `const std::string& string_name`! :)